core/src/test/java/com/radixdlt/rev2/REv2LargeTransactionTest.java

```diff
@@ -124,9 +124,10 @@ private DeterministicTest createTest() {
   }
 
   private static RawNotarizedTransaction createLargeValidTransaction() {
-    var intentBytes =
+    final var blobsSize = 1024 * 1024 - 462 /* space for other intent/notarized txn fields */;
```

Comment:
How is `462` calculated? Number of signatures * signature size + some header size?

Reply:
yeah, it's everything else. intent header and signatures